File: Breadstick/Breadstick.cpp
#include <iostream>
using namespace std;

int main()
{
    int T;
    cin >> T;

    while(T--)
    {

        int N, X;
        cin >> N >> X;

        if (N % X == 0)
        {
            cout << "YES" << endl;
        }

        else if ((N % 2 == 0 && X % 2 == 0) || X % 2 == 1)
        {
            cout << "YES" << endl;
        }
        else
        {
            cout << "NO" << endl;
        }
    }

    return 0;
}

File: Breadstick/Breadstick.py
T = int(input())
while T:
    T -= 1
    N, X = map(int, input().split())
    if (N % X == 0):
        print("YES")
    elif ((N % 2 == 0 and X % 2 == 0) or (X % 2 == 1)):
        print("YES")
    else:
        print("NO")